ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTypefaceUtils.java

```diff
@@ -85,6 +85,12 @@ public static int parseFontStyle(@Nullable String fontStyleString) {
           case "proportional-nums":
             features.add("'pnum'");
             break;
+          case "stylistic-three":
+            features.add("'ss03'");
+            break;
+          case "stylistic-five":
+            features.add("'ss05'");
+            break;
```

Comment:
Why are we adding only ss03 and ss05 here? What's your use case?
https://docs.microsoft.com/en-us/typography/opentype/spec/features_pt#ssxx

Reply:
Hi @cortinico 

Thank for your question. Because fontVariant now only have some regular setting features. In my case i would like to alternative 'a', 'g', '1' characters. It would be good if we can add all the font features. I am thinking about how dynamic it is. Do you have any advice?

Reply:
IMHO either we include all of them, or we include none.

Reply:
Hi @cortinico 
Thank you for your suggestion. I updated my pull request with fully added 20 font stylistics.